oddt/fingerprints.py

```diff
@@ -214,7 +214,7 @@ def fold(fp, size):
 def sparse_to_dense(fp, size, count_bits=True):
     """Converts sparse fingerprint (consists only 'on' bits)
     to dense (consists all bits)"""
-    sparsed_fp = np.zeros(size, dtype=np.uint8 if count_bits else bool)
+    sparsed_fp = np.zeros(size, dtype=np.uint32 if count_bits else bool)
```

Comment:
Yes, i was wrong. Idk why i thought that we use hashed atom's indices, which is certainly not true. Sorry.